# Contains all the information from ToF Sensors

# The values from middle ranged sensors
int32[4] Sensor_suite

int32 ERROR_CODE = 819